packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -39,6 +39,66 @@ export function spawnAsync(
   });
 }
 
+export function execAsync(
+  command: string,
+  args: string[],
+  opts: SpawnOptions = {}
+) {
+  return new Promise<{ stdout: string; stderr: string; code: number }>(
+    (resolve, reject) => {
+      opts.stdio = 'pipe';
+
+      let stdout: Buffer = Buffer.from('');
+      let stderr: Buffer = Buffer.from('');
+
+      const child = spawn(command, args, opts);
+
+      child.stderr!.on('data', data => {
+        stderr = Buffer.concat([stderr, data]);
```

Comment:
This is very inefficient. Instead, keep an array of Buffers and do a single `Buffer.concat()` upon the `"end"` event.